.github/workflows/get_build_targets.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# This limits the directory that bazel query is going to search under.
+readonly SEARCH_FOLDER="//source/common/..."
+
+set -e -o pipefail
+
+# This is to filter out the redudant .cc or .h build targets that bazel
+# query emits when rdeps = 0
+function filter_line() {
+  while read line
```

Comment:
Why can't this be a simple `grep`? If you have time to clean that up that would be great, otherwise we can fix later if you leave a TODO.

Reply:
Yes, you are right. I have done the change locally but still waiting for the current codeql run to finish before pushing that change.